tsconfig.json

```diff
@@ -2,24 +2,40 @@
   "compilerOptions": {
     "target": "es5",
     "module": "commonjs",
-    "esModuleInterop": true,
-    "noImplicitAny": true,
+    "jsx": "react",
```

Comment:
Shouldn't this live in the `src/core/client/tsconfig.json` file?

Reply:
This was only added for `jest`. The only alternative is to run different instances of `jest` for server and for the client. We can have a discussion around this.

Reply:
Ah I probably can make it work, without this.

Reply:
Looks like I don't need this